### PR TITLE
fix: modity issues story_point from int64 to float64

### LINKED ISSUE #4325: [Bug][Jira] Issue story points can be a float number
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

When I transformed Jira story points to the domain layer, 0.5 became 0. 

### What do you expect to happen

The number of story points should be 0.5

### How to reproduce

Collect data from Jira, make sure you set the story points configuration in the blueprint transformation.

### Anything else

_No response_

### Version

main

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by abeizn**
### Summary
modity issues story_point from int64 to float64

### Does this close any open issues?
Closes #4325 

### Screenshots

image

image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: backend/plugins/jira/models/migrationscripts/20230214_remove_issue_stdstorypoint.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/apache/incubator-devlake/core/plugin"
+)
+
+var _ plugin.MigrationScript = (*removeIssueStdStoryPoint)(nil)
+
+type removeIssueStdStoryPoint struct{}
+
+func (*removeIssueStdStoryPoint) Up(basicRes context.BasicRes) errors.Error {
+	db := basicRes.GetDal()
+	err := db.DropColumns("_tool_jira_issues", "std_story_point")
```
Comment (warren830) [inline backend/plugins/jira/models/migrationscripts/20230214_remove_issue_stdstorypoint.go:32]: why do we remove?

**Comment by warren830** [comment]
Can you also provide a screenshot after building?

**Review by warren830 — DISMISSED** [review]
LGTM

**Review by mindlesscloud — APPROVED** [review]
LGTM

**Pull request merged**